src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.BeforeCommon.targets

```diff
@@ -92,6 +92,24 @@ Copyright (c) .NET Foundation. All rights reserved.
     <Reference Include="System.Xml"/>
   </ItemGroup>
 
+  <!-- Set the default versions of the NETStandard.Library or Microsoft.NETCore.App packages to reference.
+       The implicit package references themselves are defined in Microsoft.NET.Sdk.props, so that they can be overridden
+       in the project file. -->
+  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETStandard' And '$(NetStandardDefaultPackageVersion)' == ''">
+    <NetStandardDefaultPackageVersion>1.6.0</NetStandardDefaultPackageVersion>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(NetCoreAppDefaultPackageVersion)' == ''" >
+    
+    <!-- For Microsoft.NETCore.App, default to using the version of the package that matches the target framework version -->
+    <NetCoreAppDefaultPackageVersion>$(_TargetFrameworkVersionWithoutV)</NetCoreAppDefaultPackageVersion>
+    
+    <!-- When the TFM is .NET Core 1.0, use version 1.0.1 of the Microsoft.NETCore.App package -->
+    <NetCoreAppDefaultPackageVersion Condition=" '$(_TargetFrameworkVersionWithoutV)' == '1.0' ">1.0.1</NetCoreAppDefaultPackageVersion>
```

Comment:
> This is going to be a maintenance headache. Is the plan to add conditions here for every patch release?

If we want to automatically target the latest patch release available, I think we have to have the mapping from TFM to patch release somewhere.  Do you see a better way to do this?